src/observable/BoundCallbackObservable.ts

```diff
@@ -138,11 +138,19 @@ function dispatch<T>(state: { source: BoundCallbackObservable<T>, subscriber: Su
   self.add(subject.subscribe(subscriber));
 }
 
-function dispatchNext({ value, subject }) {
+type DispatchNextArg<T> = {
```

Comment:
Ah, I thought them just as an alias, not intended to introduce a new type which are implemented from other types.

But other parts uses `interface`, I'll sort with them